sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -314,12 +314,13 @@
   color: #ff7a00;
 }
 
-.status-resolved-restriction .form-status-badge {
+.formio-success-status .form-status-badge {
```

Comment:
We have more updated status chip classes defined as `formio-status-chip` can we use those instead. They are supposed to be generic and not restriction-related.